### ISSUE TITLE
[FoxyTab] [Feature Request] Ability to sort by container

### ISSUE THREAD

**Issue by xenoruon** [erosman/support#557]
Hi, I think it would be nice to have the ability to sort by containers using a container sort order of your choosing (and the ability to set its corresponding keyboard shortcut extension).

**Comment by erosman** [comment]
It is possible, if there is a demand for it.

What about the sorting within each container?

**Comment by xenoruon** [comment]
@erosman good point!
Personally, I wouldn't care about the sorting within each container, but I guess it would also be nice to have the ability to use the other sorting options scoped to each container; maybe have another sub-menu within the "Sort" one?

- Sort
  - Global
    - All the current sorting options
    - Sort by Container Ascending
    - Sort by Container Descending
  - Container
    - All the current sorting options

I could also see the "Container" sorting options working either:
  - Sort every tab within its own container
  - Sort only the tabs in the container for the tab used to open the context menu

I guess the first option would be the way to go if using an extension hotkey shortcut though mmm

**Comment by erosman** [comment]
Next time I start to work on an update, I will see what can work out (without adding too many new menus).

**Comment by xenoruon** [comment]
@erosman Thank you for the attention and your time!

**Comment by GunGunGun** [comment]
> It is possible, if there is a demand for it.
> 
> What about the sorting within each container?

Yes, as explained in this thread, sorry for the duplicated thread I will close it asap: https://github.com/erosman/support/issues/598

**Comment by jerome-dl** [comment]
Hi there,
Just to mention that I'd be happy to have sort by container as a primary criteria for all other sorting options (or: sole criteria as well).
Thanks
